docs/source/gbd2019_models/risk_exposures/stunting/index.rst

```diff
@@ -0,0 +1,337 @@
+|       | MOD   | < -2 to -3 Z score    | 2.0 (1.2 - 3.8)    |
+|       +-------+-----------------------+--------------------+
+|       | SEV   | < -3 Z score          | 3.7 (2.0 - 8.0)    |
++-------+-------+-----------------------+--------------------+
+
+
+Vivarium Modeling Strategy
+++++++++++++++++++++++++++
+
+Child stunting in GBD has an **ordered polytomous variable**. It has rei_id 241
+
+We will use a 'propensity exposure model' for child stunting, in which each simulant is initialized with a “propensity” for a stunting category, and the simulant’s stunting category is determined by comparing this propensity to the overall stunting exposure/prevalence in the population. 
+
+.. todo::
+
+  Discuss with engineers if we want to use the propensity model or the invidivual Z-score distributions? Confirm with James how stunting was modelled in BEP.
+
+
+Restrictions
+------------
+
+.. list-table:: GBD 2019 Risk Exposure Restrictions
+   :widths: 15 15 20
+   :header-rows: 1
+
+   * - Restriction Type
+     - Value
+     - Notes
+   * - Male only
+     - False
+     -
+   * - Female only
+     - False
+     -
+   * - YLL only
+     - False
+     -
+   * - YLD only
+     - False
+     -
+   * - YLL age group start
+     - post-neonatal 
+     - 28d - 1y, age_group_id = 4 
+   * - YLL age group end
+     - 1y - 4y
+     - 1yr - 5yr age_group_id = 5
+
+..	todo::
+
+	discuss if we want to use 2020 restrictions or 2019 restrictions? The age groups differ? 
+
+.. _stunting2.2:
+
+Risk Exposure Model Diagram
+---------------------------
+
+.. image:: child_stunting_risk_hierarchy.svg
+
+
+Data Description Tables
++++++++++++++++++++++++
+
+The following code can be used to access draw-level exposure data for the child_stunting risk factor, after additionally specifying desired location, age_group, and sex IDs. Exposure category cat 1,2,3 is severe, moderate and mild and exposure category cat4 is the TMREL.
+
+.. code-block:: python
+
+  get_draws(gbd_id_type='rei_id',
+    gbd_id=241,
+    source='exposure',
+    year_id=2019,
+    gbd_round_id=6,
+    status='best',
+    location_id = [179],
+    decomp_step ='step4')
+
+The following code can be used to access the summary-exposure value.
+
+.. code-block:: python
+  
+  #get_output = go
+  stunting_sev = go(
+              "rei", 
+              rei_id= [241], 
+              measure_id=29, #summary exposure value
+              metric_id=3, #rate
+              gbd_round_id=6, 
+              location_id=[179], 
+              year_id=2019, #need to find the exposures for 2020
+              decomp_step = 'step5')
+  stunting_sev.head()
+
+  #all ages, both sexes: 0.249026 (0.279422, 0.172527)
+
+.. todo:: 
+  need to get these for 2020 
+
+Assumptions and Limitations
++++++++++++++++++++++++++++
+
+Validation Criteria
++++++++++++++++++++
+
+1. age, sex distribution of stunting prevalence categories matches GBD 2020
+2. age, sex, mean z-scores matches GBD 2020
+
```

Comment:
3. correlation between stunting and wasting matches GBD 2020

should we also include relative risks here, or will that go in the Risk Effect model document?